src/Raven.Server/Web/Studio/Processors/AbstractStudioCollectionsHandlerProcessorForPreviewRevisions.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using JetBrains.Annotations;
+using Microsoft.AspNetCore.Http;
+using Raven.Server.Documents;
+using Raven.Server.Documents.Handlers.Processors;
+using Sparrow.Json;
+
+namespace Raven.Server.Web.Studio.Processors;
+
+internal abstract class AbstractStudioCollectionsHandlerProcessorForPreviewRevisions<TRequestHandler, TOperationContext> : IDisposable
```

Comment:
I think this need to inherit from `: AbstractHandlerProcessor<TRequestHandler>`, but I see that the `AbstractStudioCollectionsHandlerProcessorForPreviewCollection` doesn't inherit as well.. 
I think we need to change them both.

Reply:
Done